assets/src/edit-story/components/panels/index.js

```diff
@@ -82,9 +85,12 @@ export function getPanels(elements) {
       { type: BACKGROUND_DISPLAY, Panel: BackgroundDisplayPanel },
       { type: BACKGROUND_OVERLAY, Panel: BackgroundOverlayPanel },
     ];
-    // If the selected element's type is video, display poster panel, too.
+    // If the selected element's type is video, display accessibility panel, too.
```

Comment:
Shouldn't the accessibility panel also be shown for background images?

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/652/commits/fb9393afca669dbd67eef52a3ca727c8d9a54de0